### DISCUSSION TITLE
Tip: MacBook Users TouchID for Security Keys

### DISCUSSION THREAD

**Discussion by JSuenram** [Ylianst/MeshCentral#3635] [Show and tell]
Just for Information:
If you enable 2FA with "SecurityKey" you can allow access to use your TouchID-Logon as 2nd factor.
So no need for separate YubiKey on MacBooks with TouchID.

<img width="883" alt="image" src="https://user-images.githubusercontent.com/68963439/153726538-0b228c0f-a808-423d-9ac7-ae6a47400b36.png">

**Comment by Ylianst** [comment]
That is a great tip. TouchID can be used as a FIDO key and I highly recommend it. It's easy to use and increases the security of your account significantly. Personally, have a second real USB FIDO key that I also register as a backup.
